enhancements/plain-k8s-bundles.md

```diff
@@ -0,0 +1,220 @@
+
+### Indexes
+
+When a `k8s+v1` bundle is added to an index, `opm` will also synthesize and add a CSV. A lot of the existing `opm` and `operator-registry` code assumes bundles always contain CSVs, which make synthesis the simplest path to supporting `k8s+v1` bundles -- in the future, we may want to revisit this decision, especially when the [`Operator` API](https://github.com/openshift/enhancements/blob/master/enhancements/olm/simplify-apis.md#proposal) is released.
+
+To fill out the dependency graph between versions of bundles, OLM considers the "replaces" field on the CSV which points to the previous version of the operator. This field will need to be supplied in the additional metadata the operator author provides in the bundle so that OLM can build dependency graphs. There is another enhancement around semantic versioning that would remove this requirement altogether, but it is independent of this enhancement.
+
+Supporting indexes with different mediatypes is a key requirement. Both `k8s+v1`  and `registry+v1` type bundles should be able to be added to an existing index. There is the potential of versioning the index itself to help differentiate between newer index images, that include the `k8s+v1` mediatype, and older versions.
+
+#### Resolving, Unpacking, and Applying
+
+At install, a CSV must be generated for a `k8s+v1` bundle so that OLM can manage the operator it represents on the cluster. It must also take care not to apply any bundle resources that are usually generated by a CSV (e.g. APIService and Service). This means that when OLM resolves and unpacks a bundle, it needs to recognize its mediatype and take appropriate action.
+
+#### Legacy Support for AppRegistry
+
+A bundle of type `k8s+v1` can be stored in an AppRegistry type catalog after synthesizing a CSV and removing resources that are usually generated by CSVs. This allows clusters with AppRegistry type catalogs to receive updates to operators whose authors have switched to publishing `k8s+v1` bundles.
+
+### User Stories
+
+#### Story 1
+
+As an operator author, Alice wants to package her operator for use with OLM. She doesn't have prior knowledge of OLM and wants to do the minimum to package her operator up.
+
+Alice has a set of kubernetes manifests describing her operator already: a single CRD, a Service, a Deployment, and some RBAC rules (a fairly straightforward operator deployment). As a user she is able to take these manifests and run an opm command to build a valid bundle. The bundle will include all the necessary metadata for OLM to manage the operator and surface it in a catalog.
+
+OLM is then able to take this bundle and install it in the cluster from a subscription to a catalog source containing the bundle. OLM installs the operator by constructing a CSV representation of the kubernetes objects and then generating an InstallPlan.
```

Comment:
Prior to this, and outside of the scope of this enhancement, Alice could use the SDK to validate the bundle for completeness using static linting provided by scorecard through the validation library.

Reply:
I think this is still going to be the case. I added a section on validation in the implementation details at @kevinrizza's suggestion.